src/p2/interpreter.rs

```diff
@@ -745,24 +752,44 @@ impl ParsedDocument {
         self.processing_imports = false;
     }
 
+    /// prints the parsed document (for debugging purposes)
+    fn show_document(&self) {
+        for section in self.sections.iter() {
+            println!(
+                "section name: {}, caption: {:?}, is_commented: {}",
+                &section.name, &section.caption, section.is_commented
+            );
```

Comment:
Use `dbg!()` for debugging, not print. It is less typing and gives more info, and it makes it obvious to everyone that it is a debug statement and not an actual requirement.

Reply:
Ohh okay